docs/user/ppl/admin/datasources.rst

```diff
@@ -77,6 +124,24 @@ Example source command with prometheus datasource ::
     >> source = my_prometheus.prometheus_http_requests_total | stats avg(@value) by job;
 
 
+Authorization of PPL commands on datasources
+==============================================
+In case of secure opensearch domains, only admins and users with roles mentioned in datasource configuration are allowed to make queries.
+For eg: with below datasource configuration, only admins and users with prometheus_access role can run queries on my_prometheus datasource. ::
```

Comment:
why admin user is special? All user map to prometheus_acess role could access my_prometheus datasource, right?

Reply:
admin has all_access role which by default should get all permissions.